lib/si-filesystem/src/lib.rs

```diff
@@ -71,6 +70,7 @@ const DIR_STR_FUNCTIONS: &str = "functions";
 const DIR_STR_SCHEMAS: &str = "schemas";
 const DIR_STR_LOCKED: &str = "locked";
 const DIR_STR_UNLOCKED: &str = "unlocked";
+const DIR_STR_BLANK_CATEGORY: &str = "__UNCATEGORIZED__";
```

Comment:
Yikes, I guess this is a possibility in that I'm guessing it's technically an optional attribute.